herbert_core/utilities/classes/@data_op_interface/unary_op_manager.m

```diff
@@ -27,8 +27,9 @@
 w = w1;
 for i=1:numel(w1)
     %----------------------------------------------------------------------
-    % The following block may be class specific
+    % The following block is overloaded by children classes not easy
+    % convertable to sigvar
```

Comment:
you should say how an individual set of lines, as opposed to a function, will be overloaded

Reply:
? not fully understand what you mean. I've replaced 'overloaded' by reimplemented, as the whoule method is overloaded but code is imlpemented differently

Reply:
reimplemented doesn't have the same problems as overloaded, so that should do